hoot-core-test/src/test/cpp/hoot/core/algorithms/LevenshteinDistanceTest.cpp

```diff
@@ -101,6 +101,6 @@ class LevenshteinDistanceTest : public HootTestFixture
 
 };
 
-CPPUNIT_TEST_SUITE_REGISTRATION(LevenshteinDistanceTest);
+CPPUNIT_TEST_SUITE_NAMED_REGISTRATION(LevenshteinDistanceTest, "quick");
```

Comment:
I believe there are 17 of these.  Just moved them to the `quick` tests explicitly.  No change in functionality just consistency.